src/Utilities/Blaze.hpp

```diff
@@ -41,4 +40,76 @@ constexpr bool useDefaultInitialization = false;
 
 // Disable HPX parallelization
 #define BLAZE_HPX_PARALLEL_MODE 0
+
+// Disable all padding
+#define BLAZE_USE_PADDING 0
```

Comment:
Does this mean Blaze will never add padding, or Blaze assumes that nothing is padded?  I'm not sure the latter is a reasonable constraint.

Reply:
This flag means Blaze won't add padding.